pages/events/index.vue

```diff
@@ -199,7 +204,17 @@ const filteredEvents = computed(() => {
     });
   }
 });
-
+function altTextSorting(event: any) {
+  if (event.title.includes("Seminar")) {
+    if (event.speaker !== "") {
+      return event.speaker + " photo";
+    } else {
+      return "Seminar speaker photo";
+    }
+  } else {
+    return event.title + " event picture";
+  }
+}
```

Comment:
If an event has the word `Seminar` on its title, this is not gonna work properly. Could you redo this function checking the type of the event instead of the title?